_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -126,7 +126,7 @@ function admin_scripts() {
 	function admin_styles() {
 		$min = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
 
-		wp_enqueue_style( 'jetpack-admin', plugins_url( "css/jetpack-admin{$min}.css", JETPACK__PLUGIN_FILE ), array( 'genericons' ), JETPACK__VERSION . '-20121016' );
+		wp_enqueue_style( 'jetpack-admin', plugins_url( "css/jetpack-admin{$min}.css", JETPACK__PLUGIN_FILE ), array( 'genericons', 'wp-components' ), JETPACK__VERSION . '-20121016' );
```

Comment:
The build will "externalize" any `@wordpress/*` imports:
https://github.com/Automattic/jetpack/blob/993a06c29253bae09eabe512339d6baf524aef92/webpack.config.js#L41

...and then automatically add them when enqueueing Jetpack admin: 

https://github.com/Automattic/jetpack/blob/c6672042e438ea5c10d4d5d46863ae7367e71e52/_inc/lib/admin-pages/class.jetpack-react-page.php#L147-L159

With that, you should already be set and not need to enqueue this here. :-) Try if it works?